open_event/static/js/admin/event/details.js

```diff
@@ -0,0 +1,7 @@
+$( document ).ready(function() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Using "FunctionExpression" is not allowed.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2650184255/issues/source?bid=3233864&fileBranchId=3324174#l1)